.github/workflows/build.yml

```diff
@@ -49,13 +49,17 @@ jobs:
             for: linux
             prepare: "debian-based"
             build_on: linux
-            use_image: ghcr.io/owtech/foundationdb-build:7.3.0-1.ow.build
-            parallel: 2
-
+            parallel: 5
+            image: foundationdb-build:7.4.0-1.ow.build
```

Comment:
We must not refer to a 7.4 image when building 7.3. Please restore the old image name.

Reply:
done